model/crud/crud.go

```diff
@@ -0,0 +1,29 @@
+// Package crud provides a crud interface
+package crud
+
+type CRUD interface {
+	// Read values
+	Read(...ReadOption) ([]Record, error)
+	// Write a value
+	Write(id string, v interface{}) error
+	// Update a record
+	Update(id string, v interface{}) error
+	// Delete a record
+	Delete(id string) error
```

Comment:
Should there be options for Write, Update, Delete even if we can't think of any yet?

```suggestion
	Write(id string, v interface{}, ...WriteOption) error
	// Update a record
	Update(id string, v interface{}, ...UpdateOption) error
	// Delete a record
	Delete(id string, ...DeleteOption) error
```